### PR TITLE
Fixed health color logic for two drive case

### PR DESCRIPTION
**Pull request by jinapurapu**
What does this do?
Fixes logic on drive health color on BasicDashboard ServerInfoCard for two drive case.

How does it look?

Screenshot from 2021-11-03 13-05-30

Fixes https://github.com/minio/console/issues/1179

### CODE REVIEW

File: portal-ui/src/screens/Console/Dashboard/BasicDashboard/ServerInfoCard.tsx [hunk 1]
```diff
@@ -89,9 +80,12 @@ const ServerInfoCard = ({ classes, server, index }: ICardProps) => {
               <span
                 className={`${classes.innerState} ${
                   activeDisks <= totalDrives / 2 && classes.redState
-                } ${
-                  activeDisks === totalDrives / 2 + 1 && classes.yellowState
-                } ${activeDisks === totalDrives && classes.greenState}`}
+                }  ${
+                  totalDrives != 2 && activeDisks === totalDrives / 2 + 1 && classes.yellowState
+                }  ${
+                  totalDrives === 2 && activeDisks === 2  && classes.greenState
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Dashboard/BasicDashboard/ServerInfoCard.tsx:86]: I think that for this line `totalDrives === 2 && activeDisks === 2` is not needed, does it work without it?

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Dashboard/BasicDashboard/ServerInfoCard.tsx (hunk 1 above)
Comment (jinapurapu) [reply portal-ui/src/screens/Console/Dashboard/BasicDashboard/ServerInfoCard.tsx]: removed

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**